src/app/containers/CpsMetadata/utils.test.js

```diff
@@ -0,0 +1,13 @@
+import { getBrandedImage } from './utils';
+
+describe('CpsMetadata utils', () => {
+  it('should return branded image', () => {
+    const actual = getBrandedImage(
+      'http://b.files.bbci.co.uk/6FC4/test/_63721682_p01kx435.jpg',
+    );
+    const expected =
+      'http://b.files.bbci.co.uk/6FC4/test/_63721682_p01kx435.jpg';
+
+    expect(actual).toEqual(expected);
```

Comment:
Should we leave this as a failing test for the moment? (instead of having actual == expected)

Reply:
That makes sense, but once I fix the issue with Chris, it wouldn't  be necessary